docs/book/user-guide/tutorials/managing-scheduled-pipelines.md

```diff
@@ -0,0 +1,332 @@
+---
+
+# Managing Scheduled Pipelines in ZenML: A Cookbook
+
+This cookbook demonstrates how to work with scheduled pipelines in ZenML through a practical example. We'll create a simple data processing pipeline that runs on a schedule, update its configuration, and finally clean up by deleting the schedule.
+
+## How Scheduling Works in ZenML
+
+ZenML doesn't implement its own scheduler but acts as a wrapper around the scheduling capabilities of supported orchestrators like Vertex AI, Airflow, Kubeflow, and others. When you create a schedule, ZenML:
+
+1. Translates your schedule definition to the orchestrator's native format
+2. Registers the schedule with the orchestrator's scheduling system
+3. Records the schedule in the ZenML metadata store
+
+The orchestrator then takes over responsibility for executing the pipeline
+according to the schedule.
+
+{% hint style="info" %}
+For our full reference documentation on schedules, see the [Schedule a Pipeline](https://docs.zenml.io/how-to/pipeline-development/build-pipelines/schedule-a-pipeline) page.
+{% endhint %}
+
+## Prerequisites
+
+Before starting this tutorial, make sure you have:
+
+1. ZenML installed and configured
+2. A supported orchestrator (we'll use [Vertex AI](https://docs.zenml.io/stacks/orchestrators/vertex) in this example)
+3. Basic understanding of ZenML pipelines and steps
+
+## Step 1: Create a Simple Pipeline
+
+First, let's create a basic pipeline that we'll schedule. This pipeline will simulate a daily data processing task.
+
+```python
+from zenml import pipeline, step
+from datetime import datetime
+
+@step
+def process_data() -> str:
+    """Simulate data processing step."""
+    return f"Processed data at {datetime.now()}"
+
+@step
+def save_results(data: str) -> None:
+    """Save processed results."""
+    print(f"Saving results: {data}")
+
+@pipeline
+def daily_data_pipeline():
+    """A simple pipeline that processes data daily."""
+    data = process_data()
+    save_results(data)
+```
+
+## Step 2: Create a Schedule
+
+Now, let's create a schedule for our pipeline. We'll set it to run daily at 9 AM.
+
+```python
+from zenml.config.schedule import Schedule
+from datetime import datetime
+
+# Create a schedule that runs daily at 9 AM
+schedule = Schedule(
+    name="daily-data-processing",
+    cron_expression="0 9 * * *"  # Run at 9 AM every day
+)
+
+# Attach the schedule to our pipeline
+scheduled_pipeline = daily_data_pipeline.with_options(schedule=schedule)
+
+# Run the pipeline to create the schedule
+scheduled_pipeline()
+```
+
+### Best Practice: Use Descriptive Schedule Names
+
+When creating schedules, follow a consistent naming pattern to better organize them:
+
+```python
+# Example of a well-named schedule
+schedule = Schedule(
+    name="daily-feature-engineering-prod-v1",
+    cron_expression="0 4 * * *"
+)
+```
+
+Include the frequency, purpose, environment, and version in your schedule names.
+
+## Step 3: Verify the Schedule
```

Comment:
We also need to verify on the GCP side. Should write a snippet / screenshot of GCP Ui to showcase this?